### PR TITLE
[hi] Localize content/en/docs/reference/scheduling/config.md

### PR DESCRIPTION
**Pull request by bishal7679**
Signed-off-by: Bishal Das <bishalhnj127@gmail.com>

Localizes `content/en/docs/reference/scheduling/config.md` to Hindi `content/hi/docs/reference/scheduling/config.md`

Fixes :- #35250

### CODE REVIEW

**Comment by bishal7679** [comment]
/assign @divya-mohan0209

__
[sema-logo](https://semasoftware.com/gh) &nbsp;**Summary:** :hammer_and_wrench: This code needs a fix

**Review by anubha-v-ardhan — COMMENTED** [review]
/retitle [hi] Localize content/en/docs/reference/scheduling/config.md

File: content/hi/docs/reference/scheduling/config.md [hunk 1]
```diff
@@ -0,0 +1,424 @@
+---
+title: सचेंडूलर कॉन्फ़िगरेशन
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:2]:
```suggestion
title: शेड्यूलर कॉन्फ़िगरेशन
```

File: content/hi/docs/reference/scheduling/config.md [hunk 2]
```diff
@@ -0,0 +1,424 @@
...
+content_type: अवधारणा
+weight: 20
+---
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+आप `kube-scheduler` के व्यवहार को एक कॉन्फ़िगरेशन फ़ाइल लिखकर और उसके पथ को कमांड लाइन आर्गुमेंटक के रूप में पारित करके अनुकूलित कर सकते हैं।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:9]:
```suggestion
आप `क्यूब-शेड्यूलर` के व्यवहार को एक कॉन्फ़िगरेशन फ़ाइल लिखकर और उसके पथ को कमांड लाइन आर्गुमेंट के रूप में पारित करके अनुकूलित कर सकते हैं।
```

File: content/hi/docs/reference/scheduling/config.md [hunk 3]
```diff
@@ -0,0 +1,424 @@
+---
+title: सचेंडूलर कॉन्फ़िगरेशन
+content_type: अवधारणा
+weight: 20
+---
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:7]:
```suggestion
{{< feature-state for_k8s_version="v1.25" state="stable" >}}
```

File: content/hi/docs/reference/scheduling/config.md [hunk 4]
```diff
@@ -0,0 +1,424 @@
...
+
+आप `kube-scheduler` के व्यवहार को एक कॉन्फ़िगरेशन फ़ाइल लिखकर और उसके पथ को कमांड लाइन आर्गुमेंटक के रूप में पारित करके अनुकूलित कर सकते हैं।
+
+<!-- overview -->
+
+<!-- body -->
+
+शेड्यूलिंग प्रोफ़ाइल आपको शेड्यूलिंग के विभिन्न चरणों को कॉन्फ़िगर करने की अनुमति देता है
+{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}} में।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:16]:
```suggestion
शेड्यूलिंग प्रोफ़ाइल आपको शेड्यूलिंग के विभिन्न चरणों को कॉन्फ़िगर करने की अनुमति देता है। 
```

File: content/hi/docs/reference/scheduling/config.md [hunk 5]
```diff
@@ -0,0 +1,424 @@
...
+
+{{< feature-state for_k8s_version="v1.19" state="beta" >}}
+
+आप `kube-scheduler` के व्यवहार को एक कॉन्फ़िगरेशन फ़ाइल लिखकर और उसके पथ को कमांड लाइन आर्गुमेंटक के रूप में पारित करके अनुकूलित कर सकते हैं।
+
+<!-- overview -->
+
+<!-- body -->
+
+शेड्यूलिंग प्रोफ़ाइल आपको शेड्यूलिंग के विभिन्न चरणों को कॉन्फ़िगर करने की अनुमति देता है
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:15]:
```suggestion
{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}} में
```

File: content/hi/docs/reference/scheduling/config.md [hunk 6]
```diff
@@ -0,0 +1,424 @@
...
+{{< glossary_tooltip text="kube-scheduler" term_id="kube-scheduler" >}} में।
+प्रत्येक चरण एक विस्तार बिंदु में उजागर होता है। प्लगइन्स शेड्यूलिंग व्यवहार प्रदान करते हैं
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:17]:
```suggestion
प्रत्येक चरण एक विस्तार बिंदु में उजागर होता है। इनमें से एक या अधिक विस्तार बिंदु को लागू करके
```

File: content/hi/docs/reference/scheduling/config.md [hunk 7]
```diff
@@ -0,0 +1,424 @@
...
+इनमें से एक या अधिक विस्तार बिंदु को लागू करके।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:18]:
```suggestion
प्लगइन्स शेड्यूलिंग व्यवहार प्रदान करते हैं।
```

File: content/hi/docs/reference/scheduling/config.md [hunk 8]
```diff
@@ -0,0 +1,424 @@
...
+
+आप `kube-scheduler --config <filename>` चलाकर शेड्यूलिंग प्रोफाइल निर्दिष्ट कर सकते हैं,
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:20]:
```suggestion
KubeSchedulerConfiguration ([v1beta2](/docs/reference/config-api/kube-scheduler-config.v1beta2/)
```

File: content/hi/docs/reference/scheduling/config.md [hunk 9]
```diff
@@ -0,0 +1,424 @@
...
+का उपयोग
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:21]:
```suggestion
या [v1beta3](/docs/reference/config-api/kube-scheduler-config.v1beta3/)) संरचना
```

File: content/hi/docs/reference/scheduling/config.md [hunk 10]
```diff
@@ -0,0 +1,424 @@
...
+KubeSchedulerConfiguration ([v1beta2](/docs/reference/config-api/kube-scheduler-config.v1beta2/)
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:22]:
```suggestion
का उपयोग करके आप `kube-scheduler --config <filename>`
```

File: content/hi/docs/reference/scheduling/config.md [hunk 11]
```diff
@@ -0,0 +1,424 @@
...
+या [v1beta3](/docs/reference/config-api/kube-scheduler-config.v1beta3/))
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:23]:
```suggestion
चलाकर शेड्यूलिंग प्रोफाइल निर्दिष्ट कर सकते हैं। 
```

File: content/hi/docs/reference/scheduling/config.md [hunk 12]
```diff
@@ -0,0 +1,424 @@
...
+संरचना।
```
Comment (divya-mohan0209) [inline content/hi/docs/reference/scheduling/config.md:24]:
```suggestion
```

**Review by divya-mohan0209 — CHANGES_REQUESTED** [review]
I'd highly recommend that this file be updated in line with the latest version available in English & then submitted for review.

However, I've left an initial set of reviews to incorporate in the first few paragraphs.

**Comment by divya-mohan0209** [comment]
@bishal7679 : Has this file been updated to be in synch with the latest English file?

**Comment by divya-mohan0209** [comment]
Hello @bishal7679 , there has been no activity on the PR for ~3 months and I'll be closing this out due to inactivity. Please feel free to reopen when you have the bandwidth/cycles to address the feedback.
/close

**Pull request closed without merging**